setup.sh

```diff
@@ -44,6 +44,9 @@ prometheis_total=1
 additional_tags = {
   "Environment" = "${ENV}"
 }
+ecs_instance_type = "t2.small"
+prom_cpu = "128"
```

Comment:
Any particular reasoning for these values out of interest?

Reply:
I haven't tried t2.micro but it might work? The numbers themselves I had to adjust as I downsized the instances to get the task to work